pyomo/contrib/viewer/model_browser.py

```diff
@@ -22,52 +22,7 @@
 
 _log = logging.getLogger(__name__)
 
-class DummyQtCore(object):
-    """
-    A dummy QtCore class to allow some testing without PyQt
-    """
-    class QModelIndex(object):
-        pass
-    class Qt(object):
-        class DisplayRole(object):
-            pass
-        class EditRole(object):
-            pass
-
-try:
-    from PyQt5 import QtCore
-except:
-    _log.exception("Cannot import PyQt5.QtCore")
-    try:
-        from PyQt4 import QtCore
-    except:
-        _log.exception("Cannot import PyQt4.QtCore")
-        class QAbstractItemModel(object):
-            def __init__(*args, **kwargs):
-                pass
-        QtCore = DummyQtCore
-    else:
-        try:
-            from PyQt4.QtGui import QAbstractItemView
-            from PyQt4.QtCore import QAbstractItemModel
-            from PyQt4 import uic
-        except:
-            _log.exception("Cannot import PyQt4")
-            class QAbstractItemModel(object):
-                def __init__(*args, **kwargs):
-                    pass
-            QtCore = DummyQtCore
-else:
-    try:
-        from PyQt5.QtWidgets import QAbstractItemView
-        from PyQt5.QtCore import QAbstractItemModel
-        from PyQt5 import uic
-    except:
-        _log.exception("Cannot import PyQt5")
-        class QAbstractItemModel(object):
-            def __init__(*args, **kwargs):
-                pass
-        QtCore = DummyQtCore
+from pyomo.contrib.viewer.pyqt_4or5 import *
```

Comment:
@eslickj: the PR is missing `pyqt_4or5`.

Reply:
Oops.  Should have all files now.